tests/5.1/default/test_task_target_default_firstprivate.c

```diff
@@ -0,0 +1,64 @@
+//------------test_task_target_default_firstprivate.c------------
```

Comment:
I think this test is for your previous PR and shouldn't be included here? If so just go ahead and delete this test 👍